packages/components/src/playground/button.stories.tsx

```diff
@@ -1,30 +1,35 @@
 import * as React from "react";
-
 import Button from "./button";
-import { action } from "@storybook/addon-actions";
```

Comment:
Component story format is nice because we no longer need to import any storybook addons. Not importing storybook stuff is important for things like `getSnapshots` and `prepareStory` to work, because non-storybook environments such as Jest won't necessarily have these addons available.